WordPress/gutenberg
# Double click block to exit zoom out mode

## What?


Adds ability to exit Zoom Out mode by double clicking on the block.

Note: the accessibility implications of click handlers are discussed in `Testing Instructions for Keyboard` below.

## Why?


It's important experiment with intuitive ways of exiting Zoom Out mode so that users can get back to edit mode.

## How?


Adds a double click handler to the block props which will exit zoom out mode if a block is clicked and we are in zoom out mode.

## Testing Instructions





- New Page
- Open Patterns inserter and select category to trigger Zoom Out mode
- Add some section patterns to the canvas
- Double click on any of the patterns and see that you leave Zoom Out mode.

### Testing Instructions for Keyboard


This action cannot be mirrored by keyboards. Therefore we have introduced an alternative means of exiting zoom out mode in https://github.com/WordPress/gutenberg/pull/64571.



## Screenshots or screencast 

https://github.com/user-attachments/assets/227347ae-44b1-4b36-8db5-7a70fee9f328
